modules/core/client/components/i18n.js

```diff
@@ -0,0 +1,30 @@
+import i18n from 'i18next';
+import { reactI18nextModule } from 'react-i18next';
+import * as translationEN from '../../../../public/locales/en/translation';
+import * as translationCS from '../../../../public/locales/cs/translation';
```

Comment:
We probably better off starting with some ISO standard codes; perhaps the same as we use for [profile languages](https://github.com/Trustroots/trustroots/tree/15a8bcebee8286eca1591f950a120420b2e08dd3/config/languages)? (ISO 639 2)

Reply:
Ok, let's use the ISO 639-2 instead of 639-1. It includes more languages and we already use it.

Reply:
Ahhh bummer, I think I've looked up something wrong here. 

I picked the ISO standard from this sentence:

>The `languages.json` is a custom made file which contains languages from `languages_orig.json` that has `iso_639_2` standard defined.

But it just states that we pick only those languages from the whole massive list and then actually in the database we use codes like `fi_FI`:

https://github.com/Trustroots/trustroots/blob/15a8bcebee8286eca1591f950a120420b2e08dd3/modules/core/client/directives/tr-languages.client.directive.js

Reply:
Would you like to revert to ISO_639-1, then? It should be as easy as dropping a commit.

Actually, ISO_639-1 will be compatible with `moment.locale()`, i18next documentation etc.
And when we need to use languages out of the scope of ISO_639-1, we can use ISO_639-3 just for them.

I would prefer not to use `fi_FI` though - not a standard, redundant information and hard to reuse in localization library configs without custom parsing.

Reply:
Sounds reasonable 👍  I'll leave it up to you to decide since you have better insight. :-) We can very easily refactor at any point if we hit issues.

Do ISO_639-1 or ISO_639-2 make difference between localized languages, such as English GB/US or French CA/FR etc? In volunteering space those are often pretty fast to crop up.

Reply:
Ah, I made some false assumptions (i.e. `fi_FI` is _not_ a redundant information). I'll research more.